wp-content/themes/invw/homepages/zones/invw-zones.php

```diff
@@ -78,10 +78,10 @@ function zone_homepagegrid() {
 	$redacted_id = 4541; // "Redacted" is a series
 	$members_id = 4514; // Members Exclusive series is known as "Sidebar" in the taxonomy
 
-	$redacted_post_query = largo_get_series_posts( $redacted_id, 1 );
-	$redacted_post = $redacted_post_query->posts[0];
-	$redacted_post->testing = 'redacted';
-	$shown_ids[] = $redacted_post->ID;
+	// $redacted_post_query = largo_get_series_posts( $redacted_id, 1 );
+	// $redacted_post = $redacted_post_query->posts[0];
+	// $redacted_post->testing = 'redacted';
+	// $shown_ids[] = $redacted_post->ID;
```

Comment:
A personal preference: multi-line comments where subsequent commented lines depend on earlier commented lines should be done as a block comment instead of a series of single-line comments. You don't need to change this, though.

```
/*
 * $redacted_post_query = largo_get_series_posts( $redacted_id, 1 );
 * $redacted_post = $redacted_post_query->posts[0];
 * $redacted_post->testing = 'redacted';
 * $shown_ids[] = $redacted_post->ID;
 */
```

Reply:
👍 good to know, thanks!